packages/bpk-component-ticket/src/BpkTicket.module.scss

```diff
@@ -41,20 +41,23 @@ $bpk-spacing-v2: true;
 }
 
 .bpk-ticket {
-  $notch-size: bpk-spacing-md();
   $ticket-padding: bpk-spacing-base();
 
   position: relative;
   display: block;
   display: flex;
   flex-direction: row;
   align-items: stretch;
+  order: 1;
+  flex: none;
+  flex-grow: 0;
```

Comment:
It is duplicated. `flex: none`  is equivalent to setting "flex: 0 0 auto".

Reply:
All right, have removed it. Thanks